Models/Slack/Events/SlackMessageEventModel.cs

```diff
@@ -22,6 +22,11 @@ public class SlackMessageEventModel :
 
         public ChatMessageModel ToChatMessageModel(IChatServiceConnection chatServiceConnection)
         {
+            // if we receive a message in a channel with only one other member, we assume it's a private message
+            var getChannelMemberCountTask = chatServiceConnection.GetMemberCountFromChannelIdAsync(SlackChannelId);
+            getChannelMemberCountTask.Wait();
```

Comment:
This will incur a lot of overhead on every message received

Reply:
I think Slack sends a distinct message type for PMs that we can key off of instead

Reply:
Or we can just `// TODO` this because who cares about Slack anymore lolol

Reply:
agree, but as mentioned in the other comment it seemed like a reasonable compromise for now until I can come up with a better way to reflect this through the shared data model for now.

Reply:
I'd prefer to not just `// TODO` it so it doesn't come back to haunt us if we ever revert back to Slack :D